src/style.css

```diff
@@ -0,0 +1,24 @@
+.body {
+  margin: 0;
+
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  width: 20px;
+  height: 16px;
```

Comment:
the stars are too wide, you may use `margin` or `gap` to make space between them, but not a `width` 😉

<img width="251" alt="image" src="https://user-images.githubusercontent.com/55802630/172788828-d712db11-edd9-4ed2-9732-1b4fd807ffe1.png">